src/main/java/SimpleCalculator.java

```diff
@@ -0,0 +1,37 @@
+public class SimpleCalculator {
+    private double firstNumber, secondNumber;
+
+
+    public void setFirstNumber(double firstNumber) {
+        this.firstNumber = firstNumber;
+    }
+
+
+    public void setSecondNumber(double secondNumber) {
+        this.secondNumber = secondNumber;
+    }
+
+    public double getAdditionResult() {
+
+        return firstNumber + secondNumber;
+    }
+
+    public double getSubtractionResult() {
+
+        return (firstNumber - secondNumber);
+    }
+
+    public double getMultiplicationResult() {
+
+        return (firstNumber * secondNumber);
```

Comment:
in this case brackets are redundant:
`return firstNumber * secondNumber;`
will be the same.